### ISSUE TITLE
show traceback on error

### ISSUE THREAD

**Issue by flying-sheep** [IRkernel/IRkernel#220]
like ipython does.

**Comment by takluyver** [comment]
Ah, I didn't realise R had a way to do this, because I'm used to Python doing it automatically on errors. It doesn't seem very complete even in plain R, though:

```
> ofop
Error: object 'ofop' not found
> traceback()
No traceback available
```

**Comment by flying-sheep** [comment]
well, because there’s no traceback here.

you need to call functions to get one :smile: 

```r
> print('a', digits='wrong')
Fehler in print.default("a", digits = "wrong") : 
  invlid 'digits' argument
> traceback()
2: print.default("a", digits = "wrong")
1: print("a", digits = "wrong")
```

**Comment by flying-sheep** [comment]
fixed via #222
